### PR TITLE
python-Jinja2: update to 2.10.1.

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: srcpkgs/python-Jinja2/template [hunk 1]
```diff
@@ -14,7 +14,7 @@ maintainer="Alessio Sergi <al3hex@gmail.com>"
 homepage="http://jinja.pocoo.org/"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-Jinja2/template]: BSD-3-Clause

File: srcpkgs/python-Jinja2/template [hunk 2]
```diff
@@ -14,7 +14,7 @@ maintainer="Alessio Sergi <al3hex@gmail.com>"
 homepage="http://jinja.pocoo.org/"
```
Comment (maxice8) [inline srcpkgs/python-Jinja2/template]: move license= to above homepage=

File: srcpkgs/python-Jinja2/template (hunk 1 above)
Comment (jnbr) [reply srcpkgs/python-Jinja2/template]: sure

**Comment by maxice8** [comment]
LGTM, it is a patchver update so i doubt python setup.py dependencies will be problematic

**Pull request merged**